testing/code/test_excinfo.py

```diff
@@ -866,14 +870,14 @@ def entry():
         excinfo = pytest.raises(ValueError, mod.entry)
         from _pytest._code.code import Code
 
-        monkeypatch.setattr(Code, "path", "bogus")
-        p = FormattedExcinfo(style="short")
-        reprtb = p.repr_traceback_entry(excinfo.traceback[-2])
-        lines = reprtb.lines
-        last_p = FormattedExcinfo(style="short")
-        last_reprtb = last_p.repr_traceback_entry(excinfo.traceback[-1], excinfo)
-        last_lines = last_reprtb.lines
-        monkeypatch.undo()
+        with monkeypatch.context() as mp:
```

Comment:
Because others use `pytester`, I think it would be worth adding a `trivial` changelog explaining this change, in case others find/have the same pattern in their test suites.

Reply:
Agreed - added a changelog.